comment_spell_check/comment_spell_check.py

```diff
@@ -322,10 +322,7 @@ def build_dictionary_list(args):
     if not isinstance(args.dict, list):
         return dict_list
 
-    for d in args.dict:
-        dpath = Path(d)
-        if dpath.exists():
-            dict_list.append(dpath)
+    dict_list.extend(args.dict)
```

Comment:
This is removing the explicit check with an implicit one. If string represents a URL, the `raise_for_status` will raise an exception if there is an issue and the `load_text_file` will raise an exception when it tries to open a non-existent file or a directory. Is there a try-except that deals with these exceptions? If not the program may "crash", exit in a very ugly manner. It is nicer to catch these, report the error and `sys.exit(1)`.

Reply:
well, at that point we don't know if it's a file path or a URL.

Reply:
The removal of this check from this location makes sense, but it wasn't replaced by a check in the location where we do know if this is a file or URL. In any case, there is a need for a try-catch so that the program exits gracefully when the URL or file don't exist and an exception is raised.